google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/startup/CloudToolsPluginInitializationComponent.java

```diff
@@ -62,6 +63,10 @@ public void initComponent() {
 
     new ConflictingAppEnginePluginCheck().notifyIfConflicting();
     new GoogleAccountPluginUninstaller().uninstallIfPresent();
+
+    if (!ApplicationManager.getApplication().isUnitTestMode()) {
+      ServiceManager.getService(CloudSdkServiceManager.class).getCloudSdkService().activate();
```

Comment:
does this mean that the Cloud SDK will start installing immediately as the user boots up the IDE?

I'm trying to recall what we decided here on doing it like this, or lazily when the user tries to run an action that needs it.

Reply:
I think we agreed on installing as soon as plugin is activated so that we don't have to block initial deploy/run action (for possibly long time)

Reply:
I'm ok with that for now.